src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts

```diff
@@ -172,9 +172,11 @@ export class TerminalTabbedView extends Disposable {
 	}
 
 	private _getLastWidgetWidth(): number {
-		const storedValue = this._storageService.get(TABS_WIDGET_WIDTH_KEY, StorageScope.WORKSPACE);
+		const widthKey = this._panelOrientation === Orientation.VERTICAL ? TABS_WIDGET_WIDTH_VERTICAL_KEY : TABS_WIDGET_WIDTH_HORIZONTAL_KEY;
+		const storedValue = this._storageService.get(widthKey, StorageScope.WORKSPACE);
+
 		if (!storedValue || !parseInt(storedValue)) {
```

Comment:
ensures the min width is used for vertical

Reply:
Let's add a comment explaining